src/main/java/com/example/accountingapp/entity/Company.java

```diff
@@ -0,0 +1,33 @@
+package com.example.accountingapp.entity;
+
+import com.sun.xml.bind.v2.TODO;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.Where;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Date;
+
+@Entity
+@NoArgsConstructor
+@Data
+@Where(clause = "is_deleted=false")
+public class Company extends BaseEntity {
+    private String title;
+    private String address1;
+    private String address2;
+    private String zip;
+    private String representative;
+    private String email;
+    @Column(columnDefinition = "DATE")
+    private LocalDateTime establishment_date;
+    private boolean enabled;
+    private String phone;
+    // TODO: 5/24/22 Rumia
```

Comment:
As I understood , state in the table but Cihat wanted us to create enums about them. Coz they are all constant...I asked them all blurry field about database to cihat...U can find all answer inside implementation document...For example there is a company status in the forms for company but not ER diagram ,  he clarified that we need a company status field as an enum ,a small change for u...

Reply:
thank you Mehmet , I saw your message in chat. Since state is not merged yet I wouldn't able to change. I will wait then when Rumia merge her part and fix accordingly